db/schema.rb

```diff
@@ -42,6 +42,7 @@
     t.datetime "updated_at", precision: 6, null: false
     t.uuid "course_year_one_id"
     t.uuid "course_year_two_id"
+    t.string "slug"
```

Comment:
I would expect this to cause some changes in the seed dump, but not 400 lines. Before we merge it, could you make sure you get latest data from develop (on old db version), migrate it to add the slug and run `rails cip_seed_dump` afterwards? I hope for like 10 line diff (just for core induction programmes) :)

Reply:
yea i'm pretty sure most of the diff is the changes you made last week but backwards - i'll update from develop at some point :)